DeepNeuralNetworks/DNN.ipynb

```diff
@@ -1,1399 +1,2548 @@
 {
```

Comment:
<p>This is probably far and away the most interesting part of this data, You have a lot of toy examples though so we may lose the students before we get to this, can we move this up in the notebook? Or at least include it in the introduction so the students know what we're working towards. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>